source/speech/__init__.py

```diff
@@ -46,6 +46,7 @@
 	RE_INDENTATION_SPLIT,
 	setSpeechMode,
 	speak,
+	speakKeyboardShortcuts,
```

Comment:
In `globalCommands.py`, `speech` is imported, not `speech.speech`. So I have thought that `speech` should be used out of the package, not `speech.speech`.
If it is not the case, I do not fully understand why `globalCommands.py` uses the variables in `speech` and not in `speech.speech`/

Anyway, if needed, I can probably use `speech.speech..speakKeyboardShortcuts` instead of `speech.speakKeyboardShortcuts` in `globalCommands.py`.

Let me know.

Reply:
>If it is not the case, I do not fully understand why globalCommands.py uses the variables in speech and not in speech.speech

I think this is because we didn't refactor usages of speech module when we performed the speech refactor. The goals was to keep it mostly backwards compatible, so internal and external code didn't need to be changed as much.

However, for simplicity, I think it's fine to keep in here.